modules/adxcgBidAdapter.js

```diff
@@ -1,95 +1,121 @@
+const BIDDER_CODE = 'adxcg';
+const SUPPORTED_AD_TYPES = [VIDEO, NATIVE];
+const SOURCE = 'pbjs10';
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: SUPPORTED_AD_TYPES,
+
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {object} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bid) {
+    return !!(bid.params.adzoneid);
+  },
+
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * an array of validBidRequests
+   * Info describing the request to the server.
+   */
+  buildRequests: function (validBidRequests, bidderRequest) {
+    utils.logMessage(`buildRequests: ${JSON.stringify(validBidRequests)}`);
+
+    // if (validBidRequests && validBidRequests.length > 0) {
+    let adZoneIds = [];
+    let prebidBidIds = [];
+    let sizes = [];
+
+    validBidRequests.forEach(bid => {
+      adZoneIds.push(utils.getBidIdParameter('adzoneid', bid.params));
+      prebidBidIds.push(bid.bidId);
+      sizes.push(utils.parseSizesInput(bid.sizes).join('|'));
+    });
 
-      let bid = bidfactory.createBid(STATUS.GOOD, bidRequest);
+    let location = utils.getTopWindowLocation();
+    let secure = location.protocol === 'https:';
+
+    let requestUrl = url.parse(location.href);
+    requestUrl.search = null;
+    requestUrl.hash = null;
+
+    // noinspection JSDeprecatedSymbols
+    let adxcgRequestUrl = url.format({
+      protocol: secure ? 'https' : 'http',
+      hostname: secure ? 'hbps.adxcg.net' : 'hbp.adxcg.net',
+      pathname: '/get/adi',
+      search: {
+        renderformat: 'javascript',
+        ver: 'r20171019PB10',
+        adzoneid: adZoneIds.join(','),
+        format: sizes.join(','),
+        prebidBidIds: prebidBidIds.join(','),
+        url: escape(url.format(requestUrl)), // encodeURIComponent or encodeURI
+        secure: secure ? '1' : '0',
+        source: SOURCE,
+        pbjs: '$prebid.version$'
+      }
+    });
 
-      bid.creative_id = adxcgBidReponse.creativeId;
-      bid.code = 'adxcg';
-      bid.bidderCode = 'adxcg';
-      bid.cpm = adxcgBidReponse.cpm;
+    return {
+      method: 'GET',
+      url: adxcgRequestUrl,
+    };
+  },
+  /**
+   * Unpack the response from the server into a list of bids.
+   *
+   * @param {*} serverResponse A successful response from the server.
+   * @return {bidRequests[]} An array of bids which were nested inside the server.
+   */
+  interpretResponse: function (serverResponse, bidRequests) {
+    // utils.logMessage(`interpretResponse serverResponse: ${JSON.stringify(response)}`); // remove for PRODUCTION
+    // utils.logMessage(`interpretResponse bidRequest ${JSON.stringify(bidRequests)}`); // remove for PRODUCTION
+    let bids = [];
+
+    serverResponse.forEach(serverResponseOneItem => {
+      let bid = {};
+
+      bid.requestId = serverResponseOneItem.bidId;
+      bid.bidderCode = spec.code;
```

Comment:
You shouldn't set the `bidderCode` here... It breaks adapter aliasing.

Our docs were probably bad at the time you wrote this, but they've since been updated.